pkg/project/project_loader.go

```diff
@@ -32,6 +33,12 @@ import (
 // it also resolves all project dependencies
 // if no -p parameter specified, then it creates a list of all projects
 func LoadProjectsToDeploy(specificProjectToDeploy string, apis map[string]api.Api, path string, fileReader util.FileReader) (projectsToDeploy []Project, err error) {
+	// projectsFolder := ""
```

Comment:
Comment from the other PR still stands. 
Please drop the commented out code, or actually put it in (as it's own commit) if it's a necessary bugfix for something :)

Reply:
done, I will add another pr just for this since this would have it's own unit test added. It's going to be a fix to support absolute paths, right now I found that that line blocks abs path usage